### PR TITLE
feat: channeling kinship boost distribution + some small features

### PR DESCRIPTION
**Pull request by dudendy**
* add `Aavegotchi Explorer` to `NavPanel`
* add sorting by `kinship` to `ClientLendings`
* better `Gotchi` channeling highlight
* channeling kinship boost distribution 
Screenshot 2022-06-25 at 20 05 22

### CODE REVIEW

**Comment by dudendy** [comment]
* add `Aavegotchi Explorer` to `NavPanel`
* add sorting by `kinship` to `ClientLendings`
* better `Gotchi` channeling highlight
* channeling kinship boost distribution 
Screenshot 2022-06-25 at 20 05 22

**Review by Vladislav94 — APPROVED** [review]

File: src/components/Gotchi/GotchiKinship/GotchiKinshitTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+];
+
+export function GotchiKinshipTooltip({ kinship }: { kinship: string }) {
+    const classes = styles();
+
+    const { isPricesLoaded, tokensPrices } = useContext<any>(TokensPricesContext);
+
+    const channelingBoots = GotchiverseUtils.countKinshipChannelingBoost(kinship);
+
+    const renderTotalChannelingPrice = (alchemica: AlchemicaList) => {
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiKinship/GotchiKinshitTooltip.tsx:26]: please add return type

File: src/components/Gotchi/GotchiKinship/GotchiKinshitTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+import { AlchemicaList } from 'shared/models/alchemica-list.model';
+import { TokensPricesContext } from 'contexts/TokensPricesContext';
+import { TokenTypes } from 'data/types';
+import { CommonUtils, GotchiverseUtils, GraphUtils } from 'utils';
+
+import { styles } from './styles';
+
+const altarsLevels = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+const tokens = [
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiKinship/GotchiKinshitTooltip.tsx:12]: please specify type for `tokens`

File: src/components/Gotchi/Gotchi.tsx [hunk 3]
```diff
@@ -28,10 +28,11 @@ interface GotchiProps {
     render: any;
     renderSvgByStats?: any;
     portal?: any;
+    lendingHighlight?: boolean;
```
Comment (dmutrokot) [inline src/components/Gotchi/Gotchi.tsx:31]: please rename to something like `isHighlightLending` to be more clear for what that property is standing for without diving deep into context

File: src/components/Gotchi/GotchiKinship/GotchiKinshitTooltip.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import { useContext } from 'react';
+import { Skeleton } from '@mui/material';
+
+import { AlchemicaList } from 'shared/models/alchemica-list.model';
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiKinship/GotchiKinshitTooltip.tsx:4]: as you have added it to models index.ts, it can be imported just from models

**Review by dmutrokot — CHANGES_REQUESTED** [review]

**Review by dmutrokot — APPROVED** [review]

**Pull request merged**